react/CozyTheme/styles.styl

```diff
@@ -41,3 +41,4 @@ $translucentWhite3=rgba(255, 255, 255, 0.16)
     --dividerColor2: $translucentWhite3
     --errorBackground: darken($errorColor, 70%)
 
+    --divider: rgba(255, 255, 255, 0.16);
```

Comment:
we already have dividerColor here too that seems to be the right one. No need to create one

Reply:
That's right, I thought it was another color